site/src/docs/patterns/navigation/building-navigation.mdx

```diff
@@ -0,0 +1,448 @@
+    <InternalLink href="/components/link">the component page.</InternalLink>
+  </li>
+  <li>
+    The learn more about the <strong>Linkbox</strong> component, please refer to
+    <InternalLink href="/components/linkbox">the component page.</InternalLink>
+  </li>
+</ul>
+
+### 6. Footer (required)
+
+The footer is placed at the bottom of the site and includes supplementary information and links, such as contact details, legal information, social media icons, and secondary navigation elements. The footer helps users find additional resources and reinforces the overall site structure by providing a consistent and accessible navigation area on every page.
+
+<PlaygroundPreview>
+  <Footer title="Helsinki Design System" footerAriaLabel="HDS Footer">
+    <Footer.Navigation navigationAriaLabel="HDS Footer navigation">
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+    </Footer.Navigation>
+    <Footer.Utilities backToTopLabel="Back to top">
+      <Footer.SoMe>
+        <Footer.Item icon={<IconFacebook />} />
+        <Footer.Item icon={<IconTwitter />} />
+        <Footer.Item icon={<IconInstagram />} />
+        <Footer.Item icon={<IconYoutube />} />
+        <Footer.Item icon={<IconTiktok />} />
+      </Footer.SoMe>
+      <Footer.Item label="Contact us" />
+      <Footer.Item label="Give feedback" />
+    </Footer.Utilities>
+    <Footer.Base copyrightHolder="Copyright" copyrightText="All rights reserved">
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+    </Footer.Base>
+  </Footer>
+</PlaygroundPreview>
+
+## Focus order & keyboard navigation
+
+Building a correct focus order is essential to ensure accessibility and provide a good user experience for all users, including those using keyboard navigation or assistive technologies. It's crucial to test your focus order using a keyboard alone to ensure it works as expected.
+
+Focus order, or tab order, refers to the sequence in which interactive elements on a webpage or application receive focus when a user navigates using a keyboard. Here's a general guideline on how to build the focus order for keyboard navigation:
+
+### Logical order
+
+Focus order should follow a logical and intuitive pattern, usually from left to right and top to bottom, mirroring the reading order in the majority of languages. This allows users to predict where the focus will go next, enhancing usability.
+
+#### Example of the order with HDS navigation elements
+
+1. Header
+2. Breadcrumbs
+3. Side navigation (including essential links if used concurrently)
+4. Tabs 
+5. Main content
+6. Footer
```

Comment:
Should we provide links here (or a separate list in the end of the page) to link all the HDS navigational componts?

Reply:
I wouldn't recommend adding links here, as this section primarily focuses on explaining the element order from a cognitive perspective. Links to the components can be found in the 'Recommended Order of Navigation Components' section at the bottom of every component. However, if we want to make these links more visible, I suggest creating a separate list at the top of the 'Recommended Order of Navigation Components' section.